ee/codegen/src/__test__/nodes/__snapshots__/api-node.test.ts.snap

```diff
@@ -116,20 +116,17 @@ class APINode(BaseAPINode):
 `;
 
 exports[`ApiNode > reject on error enabled > getNodeDisplayFile 1`] = `
-"from vellum_ee.workflows.display.nodes import BaseTryNodeDisplay, BaseAPINodeDisplay
-from uuid import UUID
+"from vellum_ee.workflows.display.nodes import BaseAPINodeDisplay, BaseTryNodeDisplay
 from ...nodes.api_node import APINode
+from uuid import UUID
 from vellum_ee.workflows.display.nodes.types import (
     NodeOutputDisplay,
     PortDisplayOverrides,
 )
 from vellum_ee.workflows.display.vellum import NodeDisplayData, NodeDisplayPosition
 
 
-class TryNodeDisplay(BaseTryNodeDisplay):
-    error_output_id = UUID("af589f73-effe-4a80-b48f-fb912ac6ce67")
-
-
+@BaseTryNodeDisplay.wrap(error_output_id=UUID("af589f73-effe-4a80-b48f-fb912ac6ce67"))
 class APINodeDisplay(BaseAPINodeDisplay[APINode]):
```

Comment:
Check out the new syntax.

This makes it so that we don't need to parameterize one of the display classes with `APINode.__wrapped_node__` manually. see `BaseTryNodeDisplay.wrap` for more